packages/fxa-auth-server/config/dev.json

```diff
@@ -43,10 +43,10 @@
     "stubs": {
       "plans": [
         {
-          "plan_id": "123doneProMonthly",
```

Comment:
Oh yeah, and you'll probably want to leave `123doneProMonthly` in here and not remove / rename it to `fortressProMonthly`